src/types.js

```diff
@@ -29,6 +29,11 @@ export type IterationKind = "key+value" | "value" | "key";
 export type SourceType = "module" | "script";
 
 export type Compatibility = "browser" | "jsc-600-1-4-17" | "node-source-maps";
+export const CompatibilityValues = {
+  BROWSER: "browser",
+  MOBILE_JSC: "jsc-600-1-4-17",
+  NODE_SOURCE_MAPS: "node-source-maps",
+};
```

Comment:
You should just make this into an array, and then we'll hard code the values in places like `src/realm.js`. No need to take on the runtime dependency on the enum object.

`"jsc-600-1-4-17"` is annoying to type but it'll likely be renamed to something more stable like `"react-native"`.